bedrock/firefox/firefox_details.py

```diff
@@ -157,28 +167,25 @@ def get_download_url(self, platform, language, version, product='firefox'):
             language = 'ja-JP-mac'
 
         if version == self.latest_version('aurora'):
-            return self._get_aurora_download_url(platform, language, version)
+            # Download links are different for localized versions
+            if language == 'en-US':
+                if platform == 'Windows':
```

Comment:
Ahh, it may be okay. Looks like there is no `win64.installer-stub.exe` on [FTP](https://ftp.mozilla.org/pub/mozilla.org/firefox/nightly/latest-mozilla-aurora/). I'll add tests anyway.